icu4c/source/i18n/unicode/usearch.h

```diff
@@ -280,9 +280,14 @@ typedef enum {
 /* open and close ------------------------------------------------------ */
 
 /**
-* Creating a search iterator data struct using the argument locale language
+* Creates a String Search iterator data struct using the argument locale language
 * rule set. A collator will be created in the process, which will be owned by
-* this search and will be deleted in <tt>usearch_close</tt>.
+* this String Search and will be deleted in <code>usearch_close</code>.
+*
+* The UStringSearch retains a pointer to both the pattern and text strings.
+* The caller must not modify or delete them while using the UStringSearch
+* is using them.
```

Comment:
I suggest you just remove this line and terminate the previous one with a period.

Reply:
Thanks!

Reply:
(I did line and the other one as well.)